src/server/api/schema.js

```diff
@@ -139,6 +139,11 @@ const rootSchema = `
     userId: String
   }
 
+  input InviteInput {
+    id: String
```

Comment:
did we need these in the input object? we're not using them, so worth testing whether we can remove them and have the input obj be empty